### PR TITLE
Support multiple project setup configurations.

### PR DESCRIPTION
**Pull request by oliverchang**
- Make the project_setup section of project.yaml be a list instead.
- Add a config_suffix option as well to append to job names.

### CODE REVIEW

File: src/appengine/handlers/cron/project_setup.py [hunk 1]
```diff
@@ -919,47 +922,46 @@ def get(self):
...
-        additional_vars=project_setup_config.get('additional_vars'))
-
-    projects_source = project_setup_config.get('source')
-    if projects_source == 'oss-fuzz':
-      projects = get_oss_fuzz_projects()
-    elif projects_source.startswith(storage.GS_PREFIX):
-      projects = get_projects_from_gcs(projects_source)
-    else:
-      raise ProjectSetupError('Invalid projects source: ' + projects_source)
+    project_setup_configs = local_config.ProjectConfig().get('project_setup')
```
Comment (inferno-chromium) [inline src/appengine/handlers/cron/project_setup.py:925]: We probably need to change the example project.yaml file in configs/test directory ? maybe add example for suffix (commented) ?

File: src/python/tests/appengine/handlers/cron/project_setup_test.py [hunk 2]
```diff
@@ -1607,41 +1623,66 @@ def setUp(self):
...
+                }
             },
-            'additional_vars': {
-                'all': {
-                    'STRING_VAR': 'VAL',
-                    'BOOL_VAR': True,
-                    'INT_VAR': 0,
+            {
+                'source': 'gs://bucket-dbg/projects.json',
+                'suffix': '_dbg',
```
Comment (inferno-chromium) [inline src/python/tests/appengine/handlers/cron/project_setup_test.py:1659]: suffix feels too generic ? maybe name_suffix or config_suffix or something else ?

Comment (oliverchang) [reply src/python/tests/appengine/handlers/cron/project_setup_test.py]: Renamed to job_suffix.

File: src/appengine/handlers/cron/project_setup.py (hunk 1 above)
Comment (oliverchang) [reply src/appengine/handlers/cron/project_setup.py:925]: Hmm, I don't think we need to document this for now. Would make it more difficult to change the format (like I'm doing now) in the future. Can add this later if there is demand.

**Pull request merged**